packages/service-library/src/servicelib/long_running_tasks/long_running_client_helper.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import redis.asyncio as aioredis
+from settings_library.redis import RedisDatabase, RedisSettings
+
+from ..redis._client import RedisClientSDK
+from .models import LRTNamespace
+
+_logger = logging.getLogger(__name__)
+
+
+class LongRunningClientHelper:
```

Comment:
Helper? are we supposed to use this everywhere?
why not just calling it `LongRunningTasksClient` ?

Reply:
No, this will only be used in services which want to call a long running task via it's RPC interface or that want to cleanup after others